cssbeautify/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for CSS Beautify v0.3.1
+// Project: https://github.com/senchalabs/cssbeautify
+// Definitions by: rictic <https://github.com/rictic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+  /**
+   * A string used for the indentation of the declaration (default is 4
+   * spaces).
+   */
+  indent?: string;
+  /**
+   * Defines the placement of open curly brace, either end-of-line (default)
+   * or separate-line
+   */
+  openbrace?: 'end-of-line'|'separate-line';
+
+  /**
+   * Always inserts a semicolon after the last ruleset(default is false)
+   */
+  autosemicolon?: boolean;
+}
+declare namespace beautify {}
```

Comment:
It should be an error in `typescript@2.0.3` to match `import * as x from "y"` with `export = function() {}` or `export = class { }`, provided that the exported value is not *also* a namespace.